src/tests/Myprofile.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import MyProfile from '../components/myprofile';
+import store from '../redux/configurestore';
+
+it('Myprofile renders correctlly', () => {
+  const tree = renderer
+    .create(<Provider store={store}><MyProfile /></Provider>)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly use React Testing Library to write tests for your Components. You can use any of these articles as guide when writing tests with React Testing Library 

1. https://www.robinwieruch.de/react-testing-library/
2. https://www.digitalocean.com/community/tutorials/how-to-test-a-react-app-with-jest-and-react-testing-library

React Testing Library is a helper DOM ([Document Object Model](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction)) library that handles interaction with components. 👍 💻